test/inductor/test_minifier.py

```diff
@@ -99,11 +99,13 @@ def inner(x):
             (test_proc.returncode, repro_proc.returncode),
         )
 
+    @unittest.skipIf(_HAS_TRITON, "openAI Triton unavailable")
```

Comment:
This would skip the test if Triton is available, so needs to be fixed unless I'm missing something